install/update_91_92.php

```diff
@@ -940,18 +941,23 @@ function update91to92() {
       $migration->addKey("glpi_savedsearches", 'last_execution_date');
    }
    //ensure do_count is set to AUTO
+   $set = ['do_count' => SavedSearch::COUNT_AUTO];
```

Comment:
```suggestion
   $set   = ['do_count' => SavedSearch::COUNT_AUTO];
```

Reply:
by the way, one question for the series of changes in this file:
Why set the `buildUpdate` parameter in variables they are not reused at all, no?

Reply:
Because the `buildQuery` method now takes parameters array as reference; the `$set` array passed as second parameter of the addPostQuery may have be changed.

Reply:
ok thank you for the explanation